### ISSUE TITLE
Constructed display cases have anti-theft protection

### ISSUE THREAD

**Issue by GravitronBlue** [tgstation/tgstation#41040]
Building a display case using wood planks and glass will by default have anti-theft protection, which when broken causes a lockdown effect in the area it's in.

My assumption is that this isn't intentional and was meant to be reserved for the captain's display case.

Currently you can build and break one basically anywhere that isn't space or lavaland and it will lock down all the doors in that area, in the past this included Reebe, but you can still abuse this to lock down the central halls or security provided you don't get caught.

TL;DR: Man with wooden planks and glass locks down entire station
